pkg/auth/dependency/interaction/provider_action.go

```diff
@@ -309,6 +296,7 @@ func (p *Provider) doTriggerOOB(i *Interaction, action *ActionTriggerOOBAuthenti
 	i.State[authenticator.AuthenticatorStateOOBOTPCode] = code
 	i.State[authenticator.AuthenticatorStateOOBOTPGenerateTime] = generateTimeStr
 	i.State[authenticator.AuthenticatorStateOOBOTPTriggerTime] = nowStr
+	i.State[authenticator.AuthenticatorPropOOBOTPChannelType] = opts.Channel
```

Comment:
In `pkg/auth/dependency/authenticator/keys.go`, the values themselves do not distinguish between Prop and State. So even I add `AuthenticatorStateOOBOTPChannelType`, the value should be the same.

Reply:
It would be confusing to maintainers to find both `AuthenticatorState` and `AuthenticatorProp` in state. I think we can do it like `AuthenticatorStateOOBOTPID` and assign the value of `AuthenticatorPropOOBOTPChannelType` to `AuthenticatorStateOOBOTPChannelType`.